cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -1101,13 +1101,8 @@ arrow::Status VeloxShuffleWriter::splitFixedWidthValueBuffer(const velox::RowVec
           binaryBuf.valueCapacity = capacity;
           dstValuePtr = binaryBuf.valuePtr + valueOffset - stringLen;
 
-          DLOG(INFO) << "Split value buffer resized. Column index: " << binaryColumnIndices_[binaryIdx] << std::endl;
-          VsPrintSplit(" dst_start", dstOffsetBase[x]);
-          VsPrintSplit(" dst_end", dstOffsetBase[x + 1]);
-          VsPrintSplit(" old size", oldCapacity);
-          VsPrintSplit(" new size", capacity);
-          VsPrintSplit(" row", partitionBufferIdxBase_[pid]);
-          VsPrintSplitLF(" string len", stringLen);
+          std::cout << "Resize column " << simpleColumnIndices_[binaryIdx] << ", partition " << pid
```

Comment:
Please use DLOG or LOG if important.

Reply:
I'm using this to analyze logs from CI, to see why the resize is triggered so frequently. These logs will be removed next.

Reply:
The old log has been changed to DLOG on main branch since #3199 merged. You should not get these logs if rebase to main.